src/process.typ

```diff
@@ -56,14 +56,16 @@
   let drawables = ()
   let bounds = none
 
-  for el in body {
-    let r = element(ctx, el)
-    if r != none {
-      if r.bounds != none {
-        bounds = aabb.aabb(r.bounds, init: bounds)
+  if body != none {
```

Comment:
Why not panic? Why pass silently?

Reply:
An empty (group) body should be legal. Example:
```typst
group({
  if ... {
    draw(...)
  }
})
```
Has the body set to `none` if the if-condition is false. This should not panic, but still create a group (or any other grouping element).

Reply:
I would prefer the check to be in group then instead of `process.many` which should always expect an array

Reply:
But isn't `none` or `()` like kind of the same? I mean, the ctx function could also return `none` which would be a second check… I favour having this at the lower level, I would even propose allowing passing functions to `process.element` and `*.many`.

Reply:
Should be solved with #491.